spec/controllers/labels_controller_spec.rb

```diff
@@ -19,22 +22,109 @@
       end
     end
 
-    context 'with valid params' do
-      before { sign_in user }
-
-      it 'returns valid json' do
+    shared_examples 'json without tasks count' do
```

Comment:
Of course, this also applies to the `include_examples`:

```suggestion
    shared_examples 'json without the number of referrencing tasks' do
```